n8n/.n8n/config

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file can be removed.  Its probably a remnant from early n8n stuff, before we changed the temp folder name from `n8n` to `.n8n`